internal/beater/beater_test.go

```diff
@@ -76,6 +77,9 @@ func TestStoreUsesRUMElasticsearchConfig(t *testing.T) {
 	)
 	require.NoError(t, err)
 	defer cancel()
+
+	time.Sleep(1 * time.Second)
```

Comment:
is this leftover from debugging?

Reply:
No, this was to ensure the metadata fetcher init was completed.

I removed the time.Sleep and used a more deterministic approach